src/sentry/eventstore/models.py

```diff
@@ -681,6 +682,30 @@ def for_group(self, group: Group):
         return GroupEvent.from_event(self, group)
 
 
+@dataclass
+class IssueEvidence:
+    name: str
+    value: str
+    # Feels like we need more attributes to allow us to decide which things to include in verbose and
+    # condensed displays. We'll use all the evidence in issue details and emails, but only important
+    # rows in slack, for example. Some thoughts:
+    # important: bool
+    # include_in: Sequence[str] # List of places this should be included in, like UI, email, slack.
+    # Maybe too configurable? Also doesn't expand well as we add more integration types. Could be
+    # more general like: UI, alerts_email, alerts_messaging, integration_ticketing, etc.
+
+
+@dataclass
+class IssueOccurrence:
+    fingerprint: Sequence[str]
+    issue_title: str
+    resource_id: str
+    evidence_data: Mapping[str, Any]
+    evidence_display: Sequence[IssueEvidence]
```

Comment:
Is this more of an explanation rather than evidence? Wondering if we should use a different term

Reply:
`IssueDetail`?

Reply:
Hmm, not sure... everything the occurrence is part of the details